accessibility/house-style.md

```diff
@@ -59,8 +59,7 @@ You may use [our VPAT template](https://github.com/springernature/vpat) when eva
 
 ## How we conform
 
-### Standards we conform to
-
+### Standards we aim for
 Our target standards are [WCAG 2.1](https://www.w3.org/TR/WCAG21/) level AA.
```

Comment:
Needs an empty line between the title and the paragraph so markdownlint doesn't complain about it 😬

Reply:
Typical 🤦‍♂️

Reply:
On the upside, happily proving my own maxim that "there is no change too small for a review".